### DISCUSSION TITLE
Suggestion

### DISCUSSION THREAD

**Discussion by andim27** [nylo-core/nylo#139] [Ideas] [answered]
Please write under release what  was removed(not only was added)

Example:
languagesList from localization.dart was removed...... and difficult find out from which release it was deleted

This approach will speed up updating projects for a new release

**Comment by agordn52** [comment, marked as answer]
Hi @andim27,

Hope you're well.
If you look through the releases, you'll find it... https://github.com/nylo-core/nylo/releases/tag/v5.25.0

The [releases](https://github.com/nylo-core/nylo/releases) page will always contain all the changes to the framework 👍
